### DISCUSSION TITLE
Bluesky users who've asked to opt-in, but their opt-in has not yet been implemented?

### DISCUSSION THREAD

**Discussion by danielhatton** [snarfed/bridgy-fed#1551] [Q&A]
Looking at the profile page of the Bluesky account \@ap.brid.gy, I notice it has 34.3k followers and 31.2k following.  Am I right in thinking the 3.1k difference represents a queue of Bluesky users who have attempted to opt in to the bridge, but whose opt-in hasn't yet been implemented?  Is there any sort of estimate of the current lead-time between attempting to opt in and the opt-in being implemented, please?

(Not a criticism - this is a brilliant service, and I understand resources are limited and the size of the task is massive.)

**Comment by snarfed** [comment]
Hi! There are lots of reasons the bridge might not currently follow someone back: they enabled the bridge but then later disabled it, their account isn't eligible (https://fed.brid.gy/docs#troubleshooting ), their username has characters we don't yet support, etc.

**Comment by danielhatton** [comment]
A development: \@ap.brid.gy has now followed back my friend on Bluesky - let's call her \@friend.bsky.social - but I still can't find an account \@friend.bsky.social‬@bsky.brid.gy to follow on the Fediverse.

**Comment by danielhatton** [comment]
Aha!  My friend's bridge account on \@bsky.brid.gy has now appeared.  I make that just over 48 hours from her following \@ap.brid.gy to \@ap.brid.gy following back, then another just over 48 hours from \@ap.brid.gy following back to her bridged account appearing.  @snarfed, do those timescales surprise you?

**Reply by snarfed** [reply]
Oof, yes, those timescales surprise me. The bridge itself does occasionally have backlogs, but usually only minutes to tens of minutes, and most of the time it's immediate. Happy to look into this if you can tell me her account.

(Bluesky itself has seen more delays on their end recently, since https://github.com/bluesky-social/atproto/discussions/3036 / https://github.com/bluesky-social/atproto/discussions/3094 , but still, 48h would be a very unusual outlier.)

**Reply by snarfed** [reply]
@danielhatton out of curiosity, how are you searching for these bridged accounts? Are you doing it on a fediverse instance that you're logged into? If you're not logged in, most fediverse instances will only show you profiles that they've already seen; they won't actually go look up a new remote user unless you're logged in.

**Reply by snarfed** [reply]
Thanks for sending over the two accounts. I see the first one followed @ap.brid.gy at 2024-11-24 20:31:11.429, the second one at 2024-11-11 11:29:26.695 and 2024-11-16 12:31:06.857 (all UTC). I do see that we had a bit of trouble sending the follow back from @ap.brid.gy - some due to Bluesky's own issues recently, https://github.com/bluesky-social/atproto/discussions/3094#discussioncomment-113856083 - but for Bluesky accounts, that follow back is just a courtesy, it has no effect on whether the profile or posts are bridged into the fediverse. Both accounts should have been visible in the fediverse as of those two initial follows.
